bundles/org.openhab.binding.solarforecast/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solarforecast"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="power-actual">
+		<item-type>Number:Power</item-type>
```

Comment:
I don't know if you saw:
https://community.openhab.org/t/solar-forecast-pv/137681/208

But if you'd like to suggest a unit here because of scale, you can use:
```suggestion
		<item-type unitHint="kW">Number:Power</item-type>
```

See for example #16621.

Of course unit should only be suggested if it makes better sense than the default (in this case W).

Reply:
Yes, I saw this comment. 
I just replied to the community post how to change the unit in the items file.
But per default I wouldn't suggest another unit than W.